heron/cli/src/python/main.py

```diff
@@ -221,21 +223,14 @@ def main():
   args, unknown_args = parser.parse_known_args()
   command_line_args = vars(args)
 
-  try:
-    if command_line_args['verbose']:
-      opts.set_verbose()
-    if command_line_args['trace_execution']:
-      opts.set_trace_execution()
-  except:
-    pass
-
   # command to be execute
   command = command_line_args['subcommand']
 
   # file resources to be cleaned when exit
   files = []
 
-  if command != 'help' and command != 'version':
+  if command not in ('help', 'version'):
+    log.set_verbose(command_line_args)
```

Comment:
Please see [`set_verbose` implementation] (https://github.com/twitter/heron/pull/1204/files/47421526742c3418cdc1b62b7b6216ce808c7f57#diff-d7d30dde4ab9b43859ead4b233f151a7R85). I agree this name is confusing. I can change to `set_logging_level`.